cmd/export_transactions.go

```diff
@@ -0,0 +1,84 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"github.com/stellar/stellar-etl/internal/input"
+	"github.com/stellar/stellar-etl/internal/transform"
+)
+
+// transactionsCmd represents the transactions command
+var transactionsCmd = &cobra.Command{
+	Use:   "export_transactions",
+	Short: "Exports the transaction data over a specified range.",
+	Long:  `Exports the transaction data over a specified range to an output file.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		startNum, endNum, limit, path := getBasicFlags(cmd)
+
+		absolutePath, err := filepath.Abs(path)
```

Comment:
This code is well-structured, so adding an option to check if path != nil should be easy, and quickly enable the stdout behavior discussed above :)